hoomd/md/external/field.py

```diff
@@ -15,12 +15,14 @@
 
 
 class Field(force.Force):
-    """Common external field potential documentation.
+    """Constructs the external field potential.
 
-    Users should not invoke `Field` directly. Documentation common to all
-    external field potentials is located here. External potentials represent
-    forces which are applied to all particles in the simulation by an external
-    agent.
+    External potentials represent forces which are applied to all particles in
+    the simulation by an external agent.
+
+    Note:
+        `Field` is the base class for all external field potentials.
+        Users should not instantiate this class directly.
```

Comment:
I tried to make this match the style of other base classes' docstrings. It felt weird that the base class previously had self-referential mentions of common documentation belonging there. It sounded like a docstring declaring itself to be a docstring.